### ISSUE TITLE
I'm a bit lost...

### ISSUE THREAD

**Issue by JOhNDooEE69** [patrickkfkan/patreon-dl#47]
So, uhh, I'm in the middle of a download spree (literally the command is running as I type) so I'm curious to see the already downloaded ones in the folder which is in System32 (right?) and I noticed that something is not right so I head straight here right away!

To make it brief, I followed and did the process (did the "Download posts by creator" format which in my case is all the 500 plus posts, which is 5 freaking hours...lol). Even the how-to obtain cookie post too, and I managed to download the posts so far, BUT the result is just the blurred image preview versions, not the clear images from the posts themselves like a subscribed Patreon user would see (yes, I saw/got the "warn: PostDownloader: Post #105950924 is not viewable by current user" for example). Did I do wrong by just copy-pasting the whole cookie directly from the header's tab straight to the command console, like is there a specific correct way to copy it (I'm using Chrome browser)? Like isn't that the whole point of inserting the cookie right? To bypass the locked posts for subscribe-only Patrons? 

Clarification would be very nice. Thanks. 

I will include some screenshot attachments below for context on what I did before I clicked Y to proceed with the download process. Sensitive info is of course redacted. (also a pic  of when I started and when I'm currently at as of writing cause why not)  
![Screenshot (7186)](https://github.com/user-attachments/assets/aa666c57-a35e-4c5a-b01c-b5c83258d473)
![Screenshot (7187)](https://github.com/user-attachments/assets/dd6a8f8f-fd42-47c6-9933-dd43a9923883)
![Screenshot (7188)](https://github.com/user-attachments/assets/2b5f9ced-5f8d-4de5-96b5-206836a9aff5)
![Screenshot (7189)](https://github.com/user-attachments/assets/444058ca-8a9a-436c-87db-c3bf449622bb)
![Screenshot (7191)](https://github.com/user-attachments/assets/ce75ba09-8604-4994-88bb-21d4a0b7171f)

Edit: ok, so just before I'm about to post, I checked again and I'm starting to get a message prompt about "MaxListernersExceededWarning" thing...that's normal right?
![Screenshot (7192)](https://github.com/user-attachments/assets/32c1c7e0-5fac-44d7-ae4c-5bfc72b820cd)
![Screenshot (7193)](https://github.com/user-attachments/assets/bd4fc064-a3ac-47fd-8512-bfa614bb54f5)
![Screenshot (7194)](https://github.com/user-attachments/assets/8dd49b92-6d9f-42ad-bd63-ce1152948ee4)
![Screenshot (7195)](https://github.com/user-attachments/assets/d6a1f415-1768-4ad7-8051-6548255f16a8)

I'm gonna rest right now, will check this post again later

**Comment by patrickkfkan** [comment]
Did you by any chance sign out of Patreon after obtaining the cookie? The point of the cookie is to give `patreon-dl` access to content viewable by you as a patron. If you sign out, the cookie will become invalid.

**Comment by JOhNDooEE69** [comment]
no I didnt, Im still logged-in at the whole time (NOT in incognito mode btw) and that was before my internet connection got unfortunately disconnected and the whole download process stopped pre-maturely. That said, the ones that are downloaded are the blurry versions, I thought that maybe the cookie that I've obtained is probably expired but I think that possibility happens if I refreshed the page after obtaining the said cookie which I didn't do of course. 

So to recap, this whole process works even if I'm not a paid subscriber to that said patron, as a patron like you said..hmmm....

I'll try the whole process again but this time to a singular post, would do a follow up comment to this after what happens.

**Comment by JOhNDooEE69** [comment]
FOLLOW-UP TO MY PREVIOUS REPLY:
man...after multiple attempts, the results are sadly still the same even for a single post, I made sure I stayed signed in for the whole session, and now I don't know what I'm missing. I guess I'll attach the last 3 attempts that I did for context. I'm looking forward to your reply.

Attempt 5: After quite a few attempts, I decided to do a fresh session and logged out of my Patreon account again, closed the window, and back on. I Log in, go straight to the post, and copy/paste a new cookie.  
![Screenshot (7203)](https://github.com/user-attachments/assets/ca6965be-370a-4290-90b8-6c77b598aef5)
![Screenshot (7206)](https://github.com/user-attachments/assets/844567c0-4ab1-4b3a-baf1-207abbc500f6)
![Screenshot (7207)](https://github.com/user-attachments/assets/4b5be95b-f109-4f36-827d-d285012bec4f)
The result is still the same as prior attempts, "warn: post #REDACTED is not viewable by current user"
...so only the preview images are downloaded, as usual...

Attempt 6: This time, I simply reloaded the page instead of the log-out/log-in strat. Copy then paste the new cookie and...
![Screenshot (7204)](https://github.com/user-attachments/assets/373f808b-c693-478d-bcf9-8c5dba3a3966)
![Screenshot (7208)](https://github.com/user-attachments/assets/0cfebef2-0307-4dcf-9657-ee6ef0ae3e03)
![Screenshot (7209)](https://github.com/user-attachments/assets/7622898f-7fab-4987-8663-761452696297)
....same result again as shown in the attachment above....

Last Attempt (for now): I reloaded the page once again for a new cookie....
![Screenshot (7205)](https://github.com/user-attachments/assets/d06637ab-8e84-4f43-a9d0-770924ae5a8a)
![Screenshot (7210)](https://github.com/user-attachments/assets/6179b9c6-6c0a-42ca-a585-76e9c666f724)
![Screenshot (7211)](https://github.com/user-attachments/assets/c9df2f79-5e0a-46b4-9e94-f11177d160b4)
....yeah...I gave up... :(

**Comment by JOhNDooEE69** [comment]
I see....so I still have to pay for their content then...anyway, I'll close this now..
